script.js

```diff
@@ -1,16 +1,27 @@
 // Write your JavaScript code here!
 
-window.addEventListener("load", function() {
+window.addEventListener("load", function () {
+    let list = document.querySelector("div[id=faultyItems]");
+    list.style.visibility = "hidden";
+    let form = document.querySelector("form");
+    let listedPlanets;
+    let listedPlanetsResponse = myFetch();
 
-   let listedPlanets;
-   // Set listedPlanetsResponse equal to the value returned by calling myFetch()
-   let listedPlanetsResponse;
-   listedPlanetsResponse.then(function (result) {
-       listedPlanets = result;
-       console.log(listedPlanets);
-   }).then(function () {
-       console.log(listedPlanets);
-       // Below this comment call the appropriate helper functions to pick a planet fom the list of planets and add that information to your destination.
-   })
-   
-});
\ No newline at end of file
+    listedPlanetsResponse.then(function (result) {
+        listedPlanets = result;
+        console.log(listedPlanets);
+    }).then(function () {
+        let planet = pickPlanet(listedPlanets);
+        addDestinationInfo(document, planet["name"], planet["diameter"], planet["star"], planet["distance"], planet["moons"], planet["image"]);
+    })
+
+    const button=document.getElementById("formSubmit");
+    button.addEventListener("click",function(e){
```

Comment:
```suggestion
    form.addEventListener("submit",function(e){
```
You can also use the `submit` event listener to listen for that event on the form itself vs. the button.